### ISSUE TITLE
NetStandard1.6 have support netCoreapp1.0 framework. Then what is the purpose of netcoreapp1.0 was given in separate?

### ISSUE THREAD

**Issue by manikandan05** [dotnet/core#500]
Hi,

i'm a beginner for ASP.NET Core and now, I have confusion with following standards.

1. NetStandard1.6
2. NetCoreApp1.0 or NetCoreApp1.1

The NetStandard1.6 will supports NetCoreApp1.0 as well as NetCoreApp1.1. But we have an options to define NetCoreApp1.0 and NetCoreApp1.1 directly in our application too. 

If NetStandard1.6 have supported this 1.0 and 1.1 means, what is the purpose of separately define netcoreapp1.0 and 1.1.

Please advise me to get out from this confusion.

Thanks.,

**Comment by Petermarcu** [comment]
.Net Core (netcoreapp1.0) is a platform as is .NET Framework (net451). Platforms have various behaviors that may not be consistent across eachother. .NET Standard is a view over multiple platforms where you can say that you want to write code against what they have in common. So if both of those platforms support .NET Standard 1.6, then you can just write a library that targets netstandard1.6 and load it in both and have high confidence it will "work everywhere".

In short, if you are building general purpose libraries, use netstandardX.X. If you are writing libraries that will only work on .NET Core because you need to depend on behaviors that are only there, use netcoreappX.X. 

In the end, we hope that only applications, not libraries need to use the platforms and all but a few special case libraries can be .NET Standard.

Hope that helps.

**Comment by manikandan05** [comment]
Hi @Petermarcu ,

Your short and crispy answer was cleared my all doubts.

Thanks.
